app/assets/stylesheets/mejs4/mejs4_add_marker_to_playlist.scss

```diff
@@ -8,6 +8,10 @@
   width: 20px;
 }
 
+.mejs__time-marker {
```

Comment:
Why is this needed in the refactor?

Reply:
Consider this an enhancement to the refactor.   It was a small style bug most likely introduced when I committed the track scrubber code, but which I never caught.  And only happens when I call a method on the MEJS4 core markers plugin directly, to "re-paint" the markers after an update.